model/jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -85,8 +85,11 @@
         
         <exclude-unlisted-classes>true</exclude-unlisted-classes>
 
+        <!-- These properties are overridden when Keycloak is managing the persistence unit by itself -->
         <properties>
             <property name="jboss.as.jpa.managed" value="false"/>
+            <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect"/>
```

Comment:
?

Reply:
This is temporary and does not really matter of WF dist. As you know, we manually create the EMF.

The reason for this is that when using a `persistence.xml`, Quarkus does not allow you to define hibernate configuration via properties file. One or another. See their docs for more details.

The idea is to also review that once we have a final solution for supporting multiple databases when using Quarkus.